Source/WebInspectorUI/UserInterface/Views/Main.css

```diff
@@ -175,6 +175,7 @@ body.docked:is(.right, .left) #navigation-sidebar.collapsed > .resizer {
 #content {
     display: grid;
     height: 100%; /* This reduces paint areas when typing in the console. http://webkit.org/b/145324 */
+    flex: 1;
```

Comment:
when did this break?  can we figure out the regression point and understand if this is something we caused or a change to something underneath us (e.g. WebCore)?  if the latter, id be concerned about more general webcompat issues

Reply:
This regressed in https://commits.webkit.org/266784@main

[The changes in `Main.css`](https://github.com/WebKit/WebKit/commit/cfbd8f5ff5d3376688299cfe672e0bba44725372#diff-0ac5431454429c67756d9554bbc2bfe142df412a37b5503ea3eb2f1b6e21f76eL177) impacted the split-view Console before. We adjusted the maximum available space for the Console Drawer in https://github.com/WebKit/WebKit/pull/27939/ to account for `min-height: 110px`  of  `#tab-browser`. But we didn't consider a multi-line console prompt in that patch.

I tested the layout with the current patch and can't notice any negative impact on the work done in https://commits.webkit.org/266784@main.

Reply:
@dcrousso Do you still have questions about this patch?